internal/code/languages.go

```diff
@@ -50,6 +51,10 @@ var Languages = map[string]Language{
 		Extension: "fish",
 		Commands:  cmds{{"fish", "<file>"}},
 	},
+	Powershell: {
+		Extension: "ps1",
+		Commands:  cmds{{"powershell", "-File", "<file>"}},
```

Comment:
```suggestion
		Commands:  cmds{{"powershell", "-NoLogo", "-NoProfile", "-File", "<file>"}},
```
You probably want to add `NoLogo`and `NoProfile` to make things faster and safer. I would also recommend using `pwsh` since that would work cross platform. I suspect the type of user who wants this tool would be using Pwsh.